patroni/ctl.py

```diff
@@ -1,4 +1,4 @@
-'''
+﻿'''
```

Comment:
There is some unwanted character:
```
@@ -1,4 +1,4 @@
-'''
+<U+FEFF>'''
 Patroni Control
 '''
```
```suggestion
'''
```